examples/calc/build.rs

```diff
@@ -1,7 +1,7 @@
 extern crate lrlex;
 
-use lrlex::process_src_dir;
+use lrlex::process_file_in_src;
 
 fn main() {
-    process_src_dir::<u8>().unwrap()
+    process_file_in_src::<u8>("calc.l", None).unwrap();
```

Comment:
I notice that the deleted line didn't have a semicolon and this one does. What does it mean for `main()` to return a value?

Reply:
Recently(ish), Rust has allowed `main` to return a `Result`, with some restrictions. Originally the `Result` returned fitting into those restrictions, but now it returns things that `main` can't handle, hence the semi-colon.

Reply:
That makes sense.